src/Transport/Receiving/MessageExtensions.cs

```diff
@@ -42,14 +41,16 @@ public static string GetMessageId(this Message message)
             return message.MessageId;
         }
 
-        public static byte[] GetBody(this Message message)
+
+        //TODO: verify if the new SDK supports sending & receiving "wcf/byte-array"
```

Comment:
@SeanFeldman I've verified this by having an old endpoint send a message to this new version and it was received correctly

`transport.BrokeredMessageBodyType(NServiceBus.Transport.AzureServiceBus.SupportedBrokeredMessageBodyTypes.ByteArray);`

Is there anything else that we might need to test?

Reply:
This was the major thing as the legacy ASB could send a serialized byte array rather than stream of bytes @andreasohlund.
Additional thing to keep in mind is that the new ASB SDK does connection pooling, where the previous client didn't do that. But you've already addressed that.

Reply:
Thanks @SeanFeldman ❤️

Reply:
np 🙂 

The cross-entity transaction API got way simpler so that only cuts out some code. Can't wait to see this refreshed transport released. And don't forget to update NSB Functions 🙂 I'll be very eager to update when it comes out.

Reply:
Yes, the functions is on our radar